authors/list_latex.py

```diff
@@ -7,7 +7,11 @@
 author_list = load(open("authors.json"))
 
 with open("../authors.tex", "wb") as f:
```

Comment:
Yeah, from what I remember, `io.open` + unicode literals is the ticket for writing Unicode in Python 2 (of course it just works in Python 3). Writing a binary file could have also worked, but it's way more messy, because you have to do all the encoding yourself.